scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/fgboost/VflGBoostAggregator.scala

```diff
@@ -12,24 +31,61 @@ import scala.collection.JavaConversions._
 import scala.collection.JavaConverters._
 import scala.collection.mutable.ArrayBuffer
 
-class VflGBoostAggregator extends Aggregator[Table] {
+class VflGBoostAggregator(validationMethods: Array[ValidationMethod[Float]] = null)
```

Comment:
FGBoostAggregator

Reply:
Solved.

Reply:
Solved.

Reply:
I found the new FGBoostAggregator is added but old VlfBoostAggregator is still there and this would fix in next refactor PR